distribution/docker/peon.sh

```diff
@@ -161,4 +161,4 @@ fi
 # If TASK_JSON is not set, CliPeon will pull the task.json file from deep storage.
 mkdir -p ${TASK_DIR}; [ -n "$TASK_JSON" ] && echo ${TASK_JSON} | base64 -d | gzip -d > ${TASK_DIR}/task.json;
 
-exec bin/run-java ${JAVA_OPTS} -cp $COMMON_CONF_DIR:$SERVICE_CONF_DIR:lib/*: org.apache.druid.cli.Main internal peon $@
+exec bin/run-java ${JAVA_OPTS} -cp $COMMON_CONF_DIR:$SERVICE_CONF_DIR:lib/*: org.apache.druid.cli.Main internal peon --taskId ${TASK_ID} $@
```

Comment:
why is this needed

Reply:
If this is absent, Druid will not be able to start the tasks, and we will receive the error log: 
ERROR [main] org.apache.druid.cli.CliPeon - Stream on task id[] is null.

This means that Druid will have problems reading the taskId. 
So, we will have to pass it in under peon.sh to ensure that we receive the task ID.